examples/single_tier_cache/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
nit. `Copyright (c) Meta Platforms, Inc. and affiliates.`

Reply:
Done